packages/core/core/src/resolveOptions.js

```diff
@@ -50,7 +50,22 @@ export default async function resolveOptions(
     entries = [path.resolve(inputCwd, initialOptions.entries)];
   }
 
-  let entryRoot = getRootDir(entries);
+  let shouldMakeEntryReferFolder = false;
+  // if entries have the only one entry and the one is not glob and refers to directory, we need to make the entry refer to some file.
```

Comment:
```suggestion
```

There's already a comment further down and also this one essentially rehashes the condition of the if.